public-docker-images/README.md

```diff
@@ -0,0 +1,52 @@
+# Veneur, in Docker
+
+This folder holds the Docker resources for building [Veneur](https://github.com/stripe/veneur), and a sample systemd unit for running it. It lives in the Veneur repository, but currently clones from Github, so the state of the repository you build in is unrelated to the output.
+
+## Building
+
+The `--no-cache` argument may be necessary to fetch the latest commits from Github, unless you're building a specific version tag.
+
+```
+public-docker-images$ docker build --no-cache -t veneur:local -f Dockerfile-debian-sid .
+```
+
+For the Alpine Linux-based image:
+```
+public-docker-images$ docker build --no-cache -t veneur:local --build-arg Dockerfile-alpine .
```

Comment:
Should this be `$ docker build --no-cache -t veneur:local -f Dockerfile-alpine .`?

Reply:
Whoops, yes!